### DISCUSSION TITLE
Conventions: Disney

### DISCUSSION THREAD

**Discussion by chenlung** [URUWorks/TeroSubtitler#265] [Ideas]
[Disney](https://disneymasteringspecs.s3.amazonaws.com/Disney_Digital_Supply_Chain_Subtitleand_CC_Style_Guide_1_1_1_2022_06_06_77ae3ac064.pdf) issued a set of guidelines for subtitles and captions and in respective languages.

Can these be added to Conventions?

**Comment by chenlung** [comment]
English has been added for the public alpha (1.0.0.0).

Adding other languages would be great. :)

**Comment by URUWorks** [comment]
v1.0.1.7

**Comment by chenlung** [comment]
I don't think all of them were added?

**Comment by GwynethLlewelyn** [comment]
> [Disney](https://disneymasteringspecs.s3.amazonaws.com/Disney_Digital_Supply_Chain_Subtitleand_CC_Style_Guide_1_1_1_2022_06_06_77ae3ac064.pdf) issued a set of guidelines for subtitles and captions and in respective languages.

BTW this link is unfortunately gone. As of 2025, the same document seems to be still available on Scribd, though: https://www.scribd.com/document/668434097/Disney-Digital-Supply-Chain-Subtitleand-CC-Style-Guide-1-1-1-2022-06-06-77ae3ac064

Grab it while you can :)
